docs/admin/hardware.rst

```diff
@@ -0,0 +1,65 @@
+
+Recommended hardware
+====================
+
+.. include:: ../includes/top-warning.rst
+
+Qubes OS hardware requirements
+------------------------------
+
+In order to install and use SecureDrop Workstation, you will need a Qubes-Compatible computer with the following speifications:
+
+- 64-bit Intel or AMD processor with virtualization support
+- a minimum of 16GB RAM (32GB recommended for production use)
+- sufficient disk space for the Qubes OS base install and SecureDrop Workstation VMs (a 128GB or greater SSD is recommended)
+
+More information on hardware compatibility can be found on the `Qubes OS System Requirements <https://www.qubes-os.org/doc/system-requirements/>`_ page, and information on specific systems can be found via the `hardware compatibility list <https://www.qubes-os.org/hcl/>`_.
+
+In order to print submissions, a supported non-networked printer is required. Supported models currently include:
+
+- TBD
+- Also TBD
+
+More printer options will be added in future releases.
+
+Lenovo ThinkPad T480
+--------------------
+The ThinkPad T480 is a viable option for SecureDrop Workstation, and also has the advantage of being used heavily by the SecureDrop development team. If you plan to use it, you should follow the instructions below to ensure that the BIOS is up to date before proceeding with the installation:
+
+.. _t480_bios:
+
+Upgrading the T480 BIOS
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The instructions below assume the use of a Linux-based computer for the creation of a BIOS upgrade USB. To upgrade the T480 BIOS:
+
+- Locate the model number of the T480 - it can often be found on a sticker on the underside of the laptop, or via the ``Main`` tab in Thinkpad Setup (accessed by pressing **Enter** on startup).
```

Comment:
"Model number" is ambiguous in the Lenovo context; "machine type" is the string they will need, which is a substring of what is called the "machine type model" in the BIOS screen. I think we should explicitly call it the "machine type" and give examples like `20L5` and `20L6`. Few things are more frustrating than trying to bash strings into the support portal search that produce no results, which is what will happen if you try to search for the full "machine type model" or other strings you may find on a sticker.

An annotated sticker like https://www.cpsc.gov/s3fs-public/styles/large/public/767_3.png?dwqT2uXDRkPddkqfw9lL4OOosQi6ht_r&itok=BTGkEBQg could be helpful, but I'm noting that my brand-new T480 straight from Lenovo has no stickers.

Reply:
:heavy_check_mark: